src/http/client.cr

```diff
@@ -866,11 +849,7 @@ class HTTP::Client
   end
 end
 
-{% if !flag?(:without_openssl) %}
-  require "openssl"
-{% end %}
 require "socket"
 require "uri"
 require "base64"
-require "./client/response"
```

Comment:
Why remove this?

Reply:
It is already provided by the `common`, along with request, headers and more.
It was just to be consistent, other than that it can be put back.